Add slurm-stats role

jovial opened the pull request.

jovial committed:
Add slurm-stats role

sjpb commented on roles/slurm-stats/README.md:
```diff
@@ -0,0 +1,32 @@
+stackhpc.slurm_openstack_tools.slurm-stats
+=========
+
+Configures slurm-stats from https://github.com/stackhpc/slurm-openstack-tools.git.
```
Maybe a nit, but could we add e.g. :

> ... which transforms `sacct` output into a form that is more amenable for importing into elasticsearch/loki.

I know we don't want to duplicate docs all over the place but equally having to chase thro 3 repos to find out what does what makes it a bit hard to use.

jovial replied:
Added your suggestion.

sjpb reviewed (changes requested):
Only one comment

jovial committed:
Merge remote-tracking branch 'ssh/main' into HEAD

jovial committed:
Switch to using ansible module to setup virtualenv

Previous code never installed slurm-openstack-tools as bin directory
was created by previous command.

jovial committed:
Remove insecure argument splatting

This is insecure as a compromised host could set arbitrary facts
and to make matters worse we are running tasks as root, see:

https://github.com/ansible/ansible/pull/43798/files#diff-c5b66da9aa38bb5a0a99a03492bd03d1cb25999de0c309bba6c6923bfe0b8d8cR470

jovial committed:
Improve summary

Addresses: https://github.com/stackhpc/ansible_collection_slurm_openstack_tools/pull/3#pullrequestreview-538258468

jovial committed:
Update defaults

sjpb reviewed (approved):
LGTM!
